src/ui/wxWidgets/PWSgrid.cpp

```diff
@@ -78,6 +79,18 @@ PWSGrid::PWSGrid(wxWindow* parent, PWScore &core,
 {
   Init();
   Create(parent, id, pos, size, style);
+  
+  // Handler for double click events on column header separator
+  auto *header = wxGrid::GetGridColHeader();
+  
+  if (header) {
+    header->Bind(
+      wxEVT_HEADER_SEPARATOR_DCLICK, 
+      [=](wxHeaderCtrlEvent& event) {
```

Comment:
Looks like you're not using any variables from the outer scope, so don't need to capture anything (`[]`).

Reply:
It's necessary to capture 'this' to be able to call the base class method 'AutoSizeColumn' within the lambda/event handler.

Reply:
I see